src/lambda/commands/copyLambdaUrl.ts

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DefaultLambdaClient, LambdaClient } from '../../shared/clients/lambdaClient'
+import { LambdaFunctionNode } from '../explorer/lambdaFunctionNode'
+import globals from '../../shared/extensionGlobals'
+import { copyToClipboard as _copyToClipboard } from '../../shared/utilities/messages'
+import { addCodiconToString } from '../../shared/utilities/textUtilities'
+import { createQuickPick, QuickPickPrompter } from '../../shared/ui/pickerPrompter'
+import { isValidResponse } from '../../shared/wizards/wizard'
+import { FunctionUrlConfigList } from 'aws-sdk/clients/lambda'
+
+export async function copyLambdaUrl(
+    node: LambdaFunctionNode,
```

Comment:
nit:

Do we have to pass in a `LambdaFunctionNode`, or could we use something simpler? Looks like the only thing that's really needed is the function name + region:
```suggestion
    node: Pick<LambdaFunctionNode, 'name' | 'regionCode'>,
```
Now for tests you can use `{ name: 'myFunction', regionCode: 'myRegion' }` instead of a stub

Reply:
Nice recommendation, I've updated to use this.